db/migrate/20170802201510_create_scholarships.rb

```diff
@@ -0,0 +1,14 @@
+class CreateScholarships < ActiveRecord::Migration[5.0]
+  def change
+    create_table :scholarships do |t|
+      t.string :name
+      t.text :description
+      t.string :location
+      t.text :terms
+      t.datetime :start_date
```

Comment:
If we want `start_date` and `end_date` to truly be dates, we should change these attributes to be of type `t.date`.  

If not, and we want them to be a `datetime`, perhaps rename the attributes to be more telling of the data they store?